example/test_driver/app_test.dart

```diff
@@ -0,0 +1,52 @@
+// Imports the Flutter Driver API.
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Webcrypto Example App', () {
+    final keyOutputFinder = find.byValueKey('KeyOutput');
+    final importKeyButtonFinder = find.byValueKey('ImportRawKey');
+    final genKeyOutputFinder = find.byValueKey('GenKeyOutput');
+    final generateKeyButtonFinder = find.byValueKey('GenerateKey');
+
+    FlutterDriver driver;
+
+    // Connect to the Flutter driver before running any tests.
+    setUpAll(() async {
+      driver = await FlutterDriver.connect();
+    });
+
+    // Close the connection to the driver after the tests have completed.
+    tearDownAll(() async {
+      if (driver != null) {
+        driver.close();
+      }
+    });
+
+    test('no imported key', () async {
+      // Verify that no key has been imported in the begining.
+      expect(await driver.getText(keyOutputFinder), "-");
+    });
+
+    test('import raw aes key', () async {
+      // Tap the button to import the key.
+      await driver.tap(importKeyButtonFinder);
+
+      // The outout should contain the importet key now.
+      expect(await driver.getText(keyOutputFinder), '{"kty":"oct","alg":"A128GCM","k":"3nle6RpFx77jwrksoNUb1Q"}');
+    });
+
+    test('no aes key generated', () async {
+      // Verify that no key has been generated in the begining.
+      expect(await driver.getText(genKeyOutputFinder), "-");
+    });
+
+    test('generate aes cbc key', () async {
+      // Tap the button to import the key.
+      await driver.tap(generateKeyButtonFinder);
+
+      // The outout should contain the importet key now.
+      expect(await driver.getText(genKeyOutputFinder), matches('{"kty":"oct","alg":"A256CBC","k":"[^"]{43}"}'));
```

Comment:
Again, it might be more robust to test:
```suggestion
expect(await driver.getText(genKeyOutputFinder),
          contains('A256CBC'));
```
(we could also parse JSON and do more accurate tests, but it's probably not necessary)

Side note: max linelength is 80 in this repo, everything is dartfmt formatted.

Reply:
Same as before, the test has been adjusted. :-)

dartfmt has also been applied.